blaze-client/src/main/scala/org/http4s/client/blaze/Http1Client.scala

```diff
@@ -20,15 +21,26 @@ object PooledHttp1Client {
       maxTotalConnections: Int = DefaultMaxTotalConnections,
       maxWaitQueueLimit: Int = DefaultMaxWaitQueueLimit,
       maxConnectionsPerRequestKey: RequestKey => Int = _ => DefaultMaxTotalConnections,
-      config: BlazeClientConfig = BlazeClientConfig.defaultConfig): Client[F] = {
+      config: BlazeClientConfig = BlazeClientConfig.defaultConfig): F[Client[F]] =
+    Effect[F].delay {
+      val http1: ConnectionBuilder[F, BlazeConnection[F]] = Http1Support(config)
+      val pool = ConnectionManager.pool(
+        http1,
+        maxTotalConnections,
+        maxWaitQueueLimit,
+        maxConnectionsPerRequestKey,
+        config.executionContext)
+      BlazeClient(pool, config, pool.shutdown())
+    }
+
+  def stream[F[_]: Effect](
```

Comment:
Can these pool properties move into the config?  It seems weird to pass this big old case class in after already having to set three properties.

Reply:
Yes